src/mme-app/handlers/s1ap_common_req_handler.c

```diff
@@ -129,7 +129,22 @@ ctx_rel_req_processing()
         return E_FAIL;
     }
 
+    char imsiStr[16] = {0};
+    imsi_bin_to_str(ue_entry->IMSI, imsiStr);
+    switch(ue_entry->ue_state)
+    {
+        case DETACH_STAGE1:
+        case DETACH_STAGE2_DS_DONE:
+        case DETACH_STAGE2_PURGE_DONE:
+        case DETACH_STAGE2:
+            log_msg(LOG_DEBUG, "Ctx Release during Detach for IMSI. Skip RABR: %s\n", imsiStr);
+            return E_FAIL;
```

Comment:
Does not look like failure. Its perfectly normal behavior right ?

Reply:
We have to skip RABR. Success will send RABR. Will update to E_SKIP.